pkgs/games/openmw/default.nix

```diff
@@ -27,6 +29,14 @@ let
         rev = "bbe61c3bc510a4f5bb4aea21cce506519c2d24e6";
         sha256 = "sha256-t3smLqstp7wWfi9HXJoBCek+3acqt/ySBYF8RJOG6Mo=";
       };
+      patches = lib.optionals stdenv.isDarwin [
```

Comment:
Does non-darwin build ok with this patch too? If so, I'd patch unconditionally - we try to avoid doing "optional" patches because when they break nobody notices.

Reply:
Most of the changes the patch makes are in areas under an `if(APPLE)` check, so it should be fine. I ran an x86_64 build with the patch and it worked, so it looks like it should be fine.